app/views/browse-solutions-page.njk

```diff
@@ -1,23 +1,29 @@
 {% extends 'includes/layout.njk' %}
-{% from 'components/hero/macro.njk' import hero %}
 {% from 'components/promo/macro.njk' import promo %}
 
 {% block body %}
+  <div class="nhsuk-width-container" data-test-id="view-solutions">
+    </br>
+    <h3>View Solutions</h3>
+    <p>
+      There are two types of Solution on the Buying Catalogue. You can choose to view Foundation Solutions only, or all that are available.
```

Comment:
is the text big enough with just using a p tag? I put class="nhsuk-u-font-size-22" on a similar tag

Reply:
Looks fine to me. Seems to match the designs.